lib/ansible/galaxy/dependency_resolution/dataclasses.py

```diff
@@ -415,13 +421,13 @@ def is_online_index_pointer(self):
 
 class Requirement(
         _ComputedReqKindsMixin,
-        namedtuple('Requirement', ('fqcn', 'ver', 'src', 'type')),
+        namedtuple('Requirement', ('fqcn', 'ver', 'src', 'type', 'signature_urls')),
```

Comment:
URLs? Maybe sources? This can be a relative file path, right?

Reply:
These should all be fixed now.